.github/workflows/main.yml

```diff
@@ -18,10 +19,9 @@ jobs:
       - name: check out diffpy.snmf
         uses: actions/checkout@v3
         with:
-          repository: diffpy/diffpy.snmf
-          # for bookkeeping have diffpy.snmf at the same level as everything else in the
-          # directory tree
+          repository: REPO_NAME
           path: .
```

Comment:
This doesn't look right, are you sure?

Reply:
Yes, that was found during my PR review and fixed probably while you were reviewing.

Reply:
I believe it should be gone when "refreshed".